apps/examples/camera/camera_main.c

```diff
@@ -449,7 +449,25 @@ void get_camera_options(int fd, int flag)
 		fprintf(stdout, SET_INTERVAL);
 	}
 }
+void print_error_log(int ctrlIndex)
```

Comment:
Is there any specific reason why added this?

Reply:
To retrieve a string and notify the developer/user about which property is failing.